src/main/java/org/minions/devfund/chelo/WhackAMole.java

```diff
@@ -0,0 +1,218 @@
+                } else {
+                    builder.append("[" + moleGrid[i][j] + "]");
+                }
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**
+     * A method to print the grid without showing where the moles are.
+     * For every spot that has recorded a whacked mole print out a W, or * otherwise.
+     * @return string builder
+     */
+    String printGrid() {
+        StringBuilder builder = new StringBuilder();
+
+        for (int i = 0; i < moleGrid.length; i++) {
+            for (int j = 0; j < moleGrid.length; j++) {
+                builder.append("[" + moleGrid[i][j] + "]");
+            }
+            builder.append("\n");
+        }
+        return builder.toString();
+    }
+
+    /**\
+     * A method to return Attempts Left value.
+     * @return remaining attempts
+     */
+    int getAttemptsLeft() {
+        return attemptsLeft;
+    }
+
+    /**
+     * A method to return moles left value.
+     * @return remaining moles
+     */
+    int getMolesLeft() {
+        return  molesLeft;
+    }
+
+    /**
+     * A method to print the current status for.
+     * Score,
+     * Attempts Left,
+     * Moles Left.
+     */
+    public void printStatus() {
+        System.out.println("Score: " + this.score
+                + ", Attempts left: " + this.attemptsLeft
+                + ", Moles left: " + this.molesLeft);
+    }
+
+    /**
+     * Main method to run the program.
+     * @param args main method
+     */
+    public static void main(final String[] args) {
+        Scanner scanner = new Scanner(System.in, "UTF-8");
+        System.out.println("Enter number of attempts:");
+        int numAttempts = scanner.nextInt(); //readUserInputOnlyIntegers();
+        System.out.println("Enter dimension for the grid:");
+        int dimensionBegin = scanner.nextInt(); //readUserInputOnlyIntegers();
+
+        WhackAMole whackAmole = new WhackAMole(numAttempts, dimensionBegin);
+
+        System.out.println("Enter the amount of moles");
+        int amountMoles = scanner.nextInt(); // readUserInputOnlyIntegers();
+        whackAmole.placeTheMoles(amountMoles, dimensionBegin);
+
+        // Printing the grid as first action to know where the moles were placed
+        System.out.println(whackAmole.printGrid());
+
+        int userInputX;
+        int userInputY;
+
+        boolean finished = false;
+
+        // While the attempts left are greater than zero the user can submit coordinates
+        while (numAttempts > 0) {
+            if (whackAmole.getMolesLeft() == 0) {
+                System.out.println("Whack-A-Mole CHAMP!!!");
+                break;
+            }
+
+            // Reading user input
+            System.out.println("Enter X: ");
+            userInputX = scanner.nextInt(); // readUserInputForGame();
+            System.out.println("Enter Y: ");
+            userInputY = scanner.nextInt(); // readUserInputForGame();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)